lib/navigation-page.php

```diff
@@ -40,6 +40,23 @@ function gutenberg_navigation_init( $hook ) {
 	);
 	$settings = gutenberg_experimental_global_styles_settings( $settings );
 
+	// Preload block editor paths.
+	// most of these are copied from edit-forms-blocks.php.
+	$preload_paths = array();
+
+	$preload_paths = apply_filters( 'navigation_editor_preload_paths', $preload_paths );
```

Comment:
For the navigation screen, I see that there was an attempt to preload certain requests in https://github.com/WordPress/gutenberg/pull/24672#issuecomment-680375627 but it didn't work well with the kind of entities managed by the navigation menu.

It looks to me that this PR could land without this part, which can be iterated later if/when we figure how to preload things for this editor.